astropy_helpers/setup_helpers.py

```diff
@@ -273,9 +269,9 @@ def run_command_hooks(cmd_obj, hook_kind):
         if isinstance(hook, str):
             try:
                 hook_obj = resolve_name(hook)
-            except ImportError as exc:
+            except ImportError:
                 raise DistutilsModuleError(
-                        'cannot find hook {0}: {1}'.format(hook, err))
+                    'cannot find hook {0}: {1}'.format(hook, err))
```

Comment:
This looks like a bug.  My guess is pylint or whatever warned that `exc` is not used, but actually it is on this line but the variable was accidentally renamed `err` here.  Should change back to `as exc` and change `err` to `exc` here.